db/seeds.rb

```diff
@@ -1,8 +1,28 @@
 # frozen_string_literal: true
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
-#   Character.create(name: "Luke", movie: movies.first)
+
+require 'faker'
+
+hall_capacities = [20, 50, 100, 200]
+
+10.times do
+  Hall.create(
+    name: Faker::FunnyName.name,
+    capacity: hall_capacities[rand(4)]
+  )
+end
+10.times do
+  Movie.create(
+    title: Faker::Movie.title,
+    description: Faker::Movie.quote,
+    duration: rand(90..189)
+  )
+end
+
+100.times do
+  Screening.create(
+    movie_id: Movie.all.sample.id,
+    hall_id: Hall.all.sample.id,
+    start_time: DateTime.current.beginning_of_minute + rand(7).days + rand(24).hours + rand(60).minutes,
```

Comment:
May something like `rand(DateTime.current .. DateTime.current + 7.days)` will be more readable?